pkg/build/api/types.go

```diff
@@ -63,6 +63,31 @@ const (
 	// forces the build to be processed by the build controller queue without waiting
 	// for a resync.
 	BuildAcceptedAnnotation = "build.openshift.io/accepted"
+
+	// BuildCreatedEvent is the reason associated with the event registered when a build is created.
+	BuildCreatedEventReason = "BuildCreated"
```

Comment:
I'd like @jwforres to weigh in, the only reason we're even creating these events is because the web console wanted to be able to convey this information to the user.